inf-clojure.el

```diff
@@ -1254,15 +1254,43 @@ See variable `inf-clojure-buffer'."
   "Return DATA if and only if it is a list."
   (when (listp data) data))
 
-(defun inf-clojure-completions (expr)
-  "Return a list of completions for the Clojure expression starting with EXPR."
+(defun inf-clojure-list-completions (expr)
```

Comment:
I don't like this name, as this function also invokes the code which generates the list it eventually processes. Not sure we won't to decouple only the part that actually processes the results. The way I see it completion has 3 stages:
* decide what's the Clojure code that will do the completion (e.g. we can set this per project or have some abilities check - try if compliment is present, fallback to `core.complete` if this fails. Probably the first option would be easier and a better initial approach to this. )
* invoke that code and get some result from it
* decide how to process the results

Reply:
Yes I think I will take the send response out of the default function. I did not notice that `inf-clojure-process-response` was there. In my mind only the string should be passed down to the custom parsing function, as you said above.